src/EnergyPlus/ScheduleManager.cc

```diff
@@ -1927,8 +1927,8 @@ namespace ScheduleManager {
 
         if (NumCommaFileShading != 0) {
             auto const &values_json = schedule_file_shading_result->second["values"];
-            auto const headers = schedule_file_shading_result->second["header"].get<std::vector<std::string>>();
-            auto const headers_set = schedule_file_shading_result->second["header"].get<std::set<std::string>>();
+            auto const headers = schedule_file_shading_result->second["header"].get<std::vector<std::string>>(); // (AUTO_OK_OBJ)
```

Comment:
I can probably add a special case to my script to capture this case. I will do that later.